code/modules/mob/living/carbon/human/human.dm

```diff
@@ -5,8 +5,11 @@
 	icon = 'icons/mob/human.dmi'
 	icon_state = "body_m_s"
 	deathgasp_on_death = TRUE
+	var/list/stamp_marks_desc = list()	// stores stamp desc
+	var/image/stamp_marks				// stores image of stamp marks
 
 /mob/living/carbon/human/New(loc)
+	stamp_marks = image('icons/effects/stamp_marks.dmi')
```

Comment:
Initially I used a list of images, then changed it to a single sprite in the form of a mutable appearance since I no longer needed seperate images for the stamp mark desc in examine. Since then I discovered overlaying images is faster than mutable appearances , so I changed it to an image. 

Although I added this define here in New() because I was getting errors trying to reference .overlays on an un instatiated variable? The overlays list didnt exist for the image here, so to get around this I gave the image an empty icon_state/icon def. This doesnt add anything visible since the `no_name` state is empty in the dmi. The check in `update_misc_effects()` isnt correctly functioning, so I will move to it check the lengeth of the overlays list of the image, and not whether or not it exists.

Reply:
Use an empty image then `stamp_marks = new /image()`